### PR TITLE
IMPLEMENTED ASCENDING PRIORITY QUEUE IN C

### LINKED ISSUE #4630: ASCENDING PRIORITY QUEUE in C
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes !

AIM :: **TO IMPLEMENT ASCENDING PRIORITY QUEUE in C.**

## Motivation

As i didn't found ASCENDING PRIORITY QUEUE in C , So i am willing to contribute it...

## Pitch

Hi I am GSSoC'21 participant,
"I am looking to work on the 

I will write the clean and optimized code , by keeping best case time and space complexity in mind and 
with  brief commenting and algorithm.
I will use C language.

### PR DESCRIPTION
**Pull request by ananddasani**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes !

### Description

I have implemented Ascending Priority Queue in C language 
so i wold request @ankitaggarwal23 , @HarshCasper to please review my PR :)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

No ! 

Fixes #4630

### CODE REVIEW

File: C/ds/AscendingPQ.c [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+WHAT IS ASCENDING PRIORITY QUEUE ?
+
+In ascending order priority queue, a lower priority number is given as a higher priority.
+For example, 1 to 5 arranged in an ascending order like,
+then, the smallest number, i.e., 1 is given as the highest priority in a Ascending priority queue.
+
+*/
+
+#include <stdio.h>
+#include <conio.h>
```
Comment (rohanbabbar04) [inline C/ds/AscendingPQ.c]: is there any use of conio.h I think you can remove it

File: C/ds/AscendingPQ.c [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+#include <stdlib.h>
+#define size 5
```
Comment (rohanbabbar04) [inline C/ds/AscendingPQ.c:14]: Also increase the size

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

File: C/ds/AscendingPQ.c (hunk 2 above)
Comment (ananddasani) [reply C/ds/AscendingPQ.c:14]: @rohanbabbar04 let PA do their work brother

Comment (HarshCasper) [reply C/ds/AscendingPQ.c:14]: @ananddasani Kindly follow the BNBR policy. If someone is asking you in good spirits to change something, you may either implement it or ask them a reason. @rohanbabbar04 has been actively reviewing some of the long pending PRs, so if not anything, kindly appreciate his efforts.

File: C/README.md [hunk 3]
```diff
@@ -185,7 +186,6 @@ _add list here_
 - [Lowest Prime Factor](math/lowest_prime_factor_of_number.c)
 - [Automorphic Number](math/automorphic_no.c)
 - [Karpekar Number](math/Kaprekar_no.c)
-- [Neon Number](math/Neon_number.c)
```
Comment (HarshCasper) [inline C/README.md:190]: Why did you delete this?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C/README.md (hunk 3 above)
Comment (ananddasani) [reply C/README.md:190]: i was showing 2 times , so

File: C/ds/AscendingPQ.c (hunk 2 above)
Comment (ananddasani) [reply C/ds/AscendingPQ.c:14]: okay :)

File: C/README.md (hunk 3 above)
Comment (ananddasani) [reply C/README.md:190]: i have added it again :)

**Review by ankitaggarwal23 — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
Hi @ananddasani 

Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!
